docs/release-notes-mtr/topics/installing-web-console-on-openshift.adoc

```diff
@@ -0,0 +1,86 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="installing-web-console-on-openshift_{context}"]
+ifdef::ocp-46[]
+= Installing the {WebName} on {ocp-short} 4.6 and later
+
+You can install the {WebName} on {ocp-short} 4.6 and later versions with the {ProductName} Operator.
+
+[NOTE]
+====
+The {ProductName} Operator is a Community Operator. Red Hat provides no support for Community Operators.
+====
+endif::[]
+ifdef::ocp-42[]
+= Installing the {WebName} on {ocp-short} 4.2-4.5
+
+You can install the {WebName} on {ocp-short} 4.2-4.5 by importing a template and instantiating it to create the {WebName} application.
+endif::[]
+
+.Prerequisites
+
+ifdef::ocp-46,ocp-42[]
+* 4 vCPUs, 8 GB RAM, and 40 GB persistent storage.
+* One or more projects in which you can install the {WebName}.
+endif::[]
++
+[IMPORTANT]
+====
+Do not install the {WebName} in a default project.
+====
+
+ifdef::ocp-46[]
+* `cluster-admin` privileges to install the {DocInfoProductName} Operator.
+* `project-admin-user` privileges to install the {WebName} application in a project.
+endif::[]
+
+ifdef::ocp-42[]
+.Procedure
+
+. Navigate to the link:{MTRDownloadPageURL}[{ProductShortName} Download page] and download the {WebName} `Local install & OpenShift` file.
+. Extract the `.zip` file to a directory, for example, `{ProductShortName}_HOME`.
+endif::[]
+ifdef::ocp-46[]
+.Installing the {DocInfoProductName} Operator
+
+. Log in to the OpenShift web console as a user with `cluster-admin` privileges.
+. Click *Operators* -> *OperatorHub*.
+. Use the *Search by keyword* field to locate the *{DocInfoProductName}* Operator.
+. Click *Install*.
+. Select a project from the *Installed Namespace* list and click *Install*.
+. Click *Operators* -> *Installed Operators* to verify that the Operator is installed.
+
+.Installing the {WebName} application
+
+. Log in to the OpenShift web console as a user with `project-admin-user` privileges.
+. Switch to the *Developer* perspective and click *+Add*.
+. In the *Add* view, click *Operator Backed*.
+. Click the *{DocInfoProductName}* Operator.
+. Click *Create*.
+endif::[]
+ifdef::ocp-42[]
+. Launch the OpenShift web console.
+. Click the *Import YAML* button in the upper-right corner of the web console.
+. Select *mta* from the *Project* list.
```

Comment:
Replace mta with mtr.

Reply:
@sshveta It's just a technical glitch - all those hundreds of files have nothing to do with the RN. They are already fixed in another PR.
Only 4 files make up the RN: 
docs/release-notes-mtr/master.adoc
docs/topics/mtr-rn-new-features.adoc
docs/topics/mtr-rn-known-issues.adoc
docs/topics/mtr-rn-resolved-issues.adoc